Update ui-class.md (#22808)
dotnet/AspNetCore.Docs: aspnetcore/razor-pages/ui-class.md

Copy the *RazorUIClassLib/Areas/MyFeature/Pages/Shared/_Message.cshtml* partial view to *WebApp1/Areas/MyFeature/Pages/Shared/_Message.cshtml*. Update the markup to indicate the new location. Build and run the app to verify the app's version of the partial is being used.

> If Razor Pages are used in the RCL, Razor Pages services and endpoints must be enabled in the hosting app:

Reviewer: ```suggestion
If the RCL uses Razor Pages, enable the Razor Pages services and endpoints in the hosting app:
```

> One must enable `Razor Pages` in the web app to make sure the page based routing works. For example, in an MVC application, register the `Razor Pages` services and map the pages with endpoint routing,

Reviewer 2: ```suggestion
If Razor Pages are used in the RCL, Razor Pages services and endpoints must be enabled in the hosting app:
```

Copy the *RazorUIClassLib/Areas/MyFeature/Pages/Shared/_Message.cshtml* partial view to *WebApp1/Areas/MyFeature/Pages/Shared/_Message.cshtml*. Update the markup to indicate the new location. Build and run the app to verify the app's version of the partial is being used.

One must enable `Razor Pages` in the web app to make sure the page based routing works. For example, in an MVC application, register the `Razor Pages` services and map the pages with endpoint routing,

> ```csharp

Reviewer 2: Finish moving this to file. Use snippet syntax to highlight the two lies
Example syntax
`[!code-csharp[](start-mvc/sample/src/MvcMovie/Models/MovieDateFormat.cs?name=snippet1&highlight=2,10)]`
